patches/react-native-fast-crypto+2.0.0.patch

```diff
@@ -0,0 +1,20 @@
+diff --git a/node_modules/react-native-fast-crypto/android/build.gradle b/node_modules/react-native-fast-crypto/android/build.gradle
+index dd999fa..79db185 100644
+--- a/node_modules/react-native-fast-crypto/android/build.gradle
++++ b/node_modules/react-native-fast-crypto/android/build.gradle
+@@ -47,15 +47,6 @@ android {
+             path "src/main/cpp/CMakeLists.txt"
+         }
+     }
+-
+-    // If you want Gradle to package prebuilt native libraries
+-    // with your APK, modify the default source set configuration
+-    // to include the directory of your prebuilt .so files as follows.
+-    sourceSets {
+-        main {
+-            jniLibs.srcDirs 'jni/libs/'
+-        }
+-    }
+ }
```

Comment:
What's the context around this patch?

Reply:
i had mentioned in the description (patch react-native-fast-crypto which is not compatible with gradle 4 yet) but noticed i only linked the issue - this is the real reason https://developer.android.com/studio/releases/gradle-plugin#cmake-imported-targets - basically this block is conflicting with gradle version 4 and should be removed

Reply:
My bad missed this 👍